hotfixes/hotfix-sp-repair

```diff
@@ -141,15 +141,74 @@ repairBSLServices() {
 	fi
 }
 
+repairNomadParentContainer() {
+	echo running repairNomadParentContainer
+	npc_image_local="nomad-parent-container:local"
+	npc_image_backup="nomad-parent-container:backup"
+
+	if [[ $(docker images --format "{{.ID}}" $npc_image_local | wc -l) -gt 0 ]]; then
+		echo "$npc_image_local already installed. Skipping to repair $npc_image_local..."
```

Comment:
"Skipping to repair" does not make sense. Could we change it to "Skipping repair of" instead?